[Jules]: fix guest-access-card GroupPolicy type

### Agent Prompt

<role>
You are a Senior Frontend Engineer developing Custom Cards for Home Assistant using Lit and TypeScript. Our CI/CD pipeline is failing the `npx tsc --noEmit` step with `error TS2305: Module '"./types/meraki"' has no exported member 'GroupPolicy'`.
</role>

<context>
The `meraki-guest-access-card.ts` file is attempting to import a `GroupPolicy` type that doesn't exist in our `types/meraki.ts` definitions file. We need to either properly define this missing type or remove the import if it's a leftover artifact.
</context>

<instructions>
1. Think step-by-step. Analyze `src/meraki-guest-access-card.ts` to determine if `GroupPolicy` is actively being used for a property, method, or state variable inside <thinking> tags.
2. Check the requirements to prevent regressions and update requirements based on implemented features.
3. Action 1: If `GroupPolicy` is actively used, open `src/types/meraki.ts` and export a new `GroupPolicy` interface. (A standard Meraki Group Policy type should include at least `groupPolicyId: string` and `name: string`).
4. Action 2: If `GroupPolicy` is NOT used in `meraki-guest-access-card.ts`, simply remove `GroupPolicy` from the import statement at the top of the file.
5. Always output the complete merged files for any files you modify so we can proactively pass the TypeScript compilation step.
</instructions>

<examples>
Example 1: Exporting the missing type
Positive Alternative: If the card relies on the type, ensure it is exported correctly in the types file.

```typescript
// In src/types/meraki.ts
export interface GroupPolicy {
  groupPolicyId: string;
  name: string;
}
</examples>

--- a/frontend/src/types/meraki.ts
+++ b/frontend/src/types/meraki.ts
@@ -11,3 +11,9 @@ export interface SSID {
   networkId: string;
   authMode?: string;
 }
+
+export interface GroupPolicy {
+  networkId: string;
+  groupPolicyId: string;
+  name: string;
+}

--- a/frontend/src/utils/meraki-data.ts
+++ b/frontend/src/utils/meraki-data.ts
@@ -1,14 +1,8 @@
 // src/utils/meraki-data.ts
 import { HomeAssistant } from '../types/ha';
-import { Network, SSID } from '../types/meraki';
+import { Network, SSID, GroupPolicy } from '../types/meraki';
 import { WsCommand } from '../types/websocket';
 import { safeCallWS } from './api';
-
-export interface GroupPolicy {
-  networkId: string;
-  groupPolicyId: string;
-  name: string;
-}
 
 export class MerakiDataProvider {
   /**

